02.calendar/cal.rb

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env ruby
+
+require "date"
+require "optparse"
+
+# 引数を読み込む
+opt = OptionParser.new
+option = {}
+opt.on("-y") {|x| option[:y] = x}
+opt.on("-m") {|x| option[:m] = x}
+opt.parse!(ARGV)
+
+# 引数チェック
+if option[:y] && option[:m]
+    year = ARGV[0].to_i
```

Comment:
```suggestion
  year = ARGV[0].to_i
```

Rubyは1インデントに付き＝スペース2文字とすることが多いです。
エディタの設定を見直してください〜。

Reply:
VSCodeのタブの設定を2文字に変更いたしました。